analysis/grammar_search_analysis.py

```diff
@@ -24,14 +24,34 @@ def _run_proxy_analysis(env_names: List[str], score_function_names: List[str],
     utils.update_config({"seed": 0})
     if "cover" in env_names:
         env_name = "cover"
-        HandEmpty, Holding = _get_predicates_by_names(env_name,
-                                                      ["HandEmpty", "Holding"])
+        HandEmpty, Holding, Covers = _get_predicates_by_names(
+            env_name, ["HandEmpty", "Holding", "Covers"])
+        targ_type = Covers.types[1]
         NotHandEmpty = HandEmpty.get_negation()
+
+        def _Clear_holds(state: State, objects: Sequence[Object]) -> bool:
```

Comment:
any reason to implement this this way instead of using Covers? i'm guessing it's equivalent to the feature that will be learned by the grammar, but good to be sure

Reply:
i wanted to avoid the pylint issue of `CoverEnv._Covers_holds` but on second thought, it's cleaner to do it that way, so will fix